deploy/examples/create-external-cluster-resources.py

```diff
@@ -1789,6 +1779,26 @@ def gen_json_out(self):
             },
         ]
 
+        if self._arg_parser.config_file:
+            json_out.append(
+                {
+                    "name": "external-cluster-user-command",
+                    "kind": "ConfigMap",
+                    "data": {
+                        "args": self.out_map["ARGS"],
+                    },
+                },
+            )
+        else:
+            json_out.append(
+                {
+                    "name": "external-cluster-user-command",
+                    "kind": "ConfigMap",
+                    "data": {
+                        "command": self.out_map["EXTERNAL_CLUSTER_USER_COMMAND"],
+                    },
+                },
+            )
```

Comment:
I'd propose that we always just output the config file. Then, in user docs, we instruct users to copy the config to a file, and then use that file as the `--config` arg. Then the user instructions are simplest to document and follow.

Reply:
This we will force user to use config.ini with upgrades.
I think if user has used config.ini at start then only give it at the next upgrade..